cavern/src/main/java/org/opencadc/cavern/FileSystemNodePersistence.java

```diff
@@ -111,23 +109,18 @@ public class FileSystemNodePersistence implements NodePersistence {
     private final PosixMapperClient posixMapper;
 
     private final Path root;
-
     private final MultiValuedProperties config;
-    
-    
 
     public FileSystemNodePersistence() {
         CavernConfig cavernConfig = new CavernConfig();
         this.config = cavernConfig.getConfig();
-        String rootConfig = config.getFirstPropertyValue(CavernConfig.FILESYSTEM_BASE_DIR);
-        if (rootConfig == null) {
-            throw new RuntimeException("CONFIG: Failed to find " + CavernConfig.FILESYSTEM_BASE_DIR);
-        }
-        this.root = Paths.get(rootConfig);
-        
+        String baseDir = config.getFirstPropertyValue(CavernConfig.FILESYSTEM_BASE_DIR);
+        String subPath = config.getFirstPropertyValue(CavernConfig.FILESYSTEM_SUB_PATH);
+        this.root = Paths.get(baseDir + subPath);
+
         // must be hard coded to this and not set via java system properties
         this.identityManager = new PosixIdentityManager();
-        
+
         LocalAuthority loc = new LocalAuthority();
         // TODO: move constant to cadc-registry
         URI posixMapperID = loc.getServiceURI("http://www.opencadc.org/std/posix#group-mapping-1.0");
```

Comment:
I have fixed this in my current code to:
```
URI posixMapperID = loc.getServiceURI(Standards.POSIX_GROUPMAP.toASCIIString());
```
but not pushed yet.

Reply:
Updated my copy.